deluge/tests/common.py

```diff
@@ -150,6 +168,26 @@ def errReceived(self, data):  # NOQA
 
 def start_core(listen_port=58846, logfile=None, timeout=10, timeout_msg=None,
                custom_script="", print_stderr=True, extra_callbacks=None):
+    """
+    Start the deluge core as a daemon
+
+    Kwargs:
```

Comment:
I was looking here which uses Kwargs as well as Args: https://pythonhosted.org/an_example_pypi_project/sphinx.html#full-code-example

But maybe that's not necessary.